_posts/2023-05-11-Discontinuing-custom-Logstash-distribution.md

```diff
@@ -0,0 +1,59 @@
+To help OpenSearch reach more people, we provide two Logstash plugins so that OpenSearch users can ingest data to and from OpenSearch using Logstash. 
+Those two plugins are [logstash-output-plugin](https://github.com/opensearch-project/logstash-output-opensearch) and [logstash-input-plugin](https://github.com/opensearch-project/logstash-input-opensearch).  
+We have also provided a custom distribution of Logstash with these plugins embedded.
+However, we believe that OpenSearch users will continue to get as much benefit from the two plugins using the Apache-licensed Logstash distribution from Elastic. 
+Therefore, we plan to discontinue the custom Logstash distribution by October 2023.
+
+## Reasons to discontinue
+
+Many users are downloading the custom OpenSearch distribution of Logstash, and we acknowledge that it does provide some convenience and value to OpenSearch users. 
+However, we also recognize some compelling reasons to discontinue the distribution.
+
+One of the reasons is to help users of these plugins get updates quickly. 
+Because the custom distribution is based on the Apache-licensed distribution provided by Elastic, users cannot get the updates immediately after Elastic releases them. 
+Instead, they have to wait for the OpenSearch project to repackage the updates.
+This is especially important for security patches, when getting these updates as soon as possible is a consideration.
+
+Another reason is that, by discontinuing the distribution, it becomes perfectly clear to users who exactly is responsible for supporting the tools they use.  
+When a user needs support for Logstash features or bundled Logstash plugins, they typically go to the [Logstash discuss](https://discuss.elastic.co/c/elastic-stack/logstash/14) page.
+When they have an issue with the OpenSearch plugins, however, they know to seek support from the OpenSearch project.
+
+Finally, the main Logstash distribution already supports downloads for a variety of platforms. 
+For example, it supports Linux, macOS, and RPM installations.
+The custom OpenSearch distribution, on the other hand, lacks many of these distributions.
+
+Installing the OpenSearch plugins in the main Logstash distribution is straightforward.
+And given the ease of installation, along with the reasons we've already listed, we believe that a decision to discontinue the distribution outweighs the convenience associated with it.  
+Discontinuing this distribution will also free up time and resources so we can better focus on delivering great features in our core products and the Logstash plugins.
+
+## Going forward
+
+The OpenSearch project will continue to maintain the existing Logstash plugins for OpenSearch.
+To install the plugins, first [download Logstash](https://www.elastic.co/downloads/logstash) from Elastic and then run the following commands.
+
+For the logstash-output-opensearch plugin, run:
```

Comment:
Above, this is "logstash-output-plugin". Is this something different?

Reply:
I will correct this above. It should be `logstash-output-plugin`.